lib/components/nav_drawer_widget.dart

```diff
@@ -0,0 +1,142 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:pace_app/app/app.dart';
+
+import '../navigation_drawer/navigation_drawer.dart';
+
+class NavDrawerWidget extends StatelessWidget {
+  final String accountEmail;
+
+  final List<_NavigationItem> _listItems = [
+    _NavigationItem(null, null, null, type: NavigationItemType.header),
+    _NavigationItem(NavItem.home, "Home", Icons.home),
+    _NavigationItem(null, null, null, type: NavigationItemType.divider),
+    _NavigationItem(NavItem.settings, "Settings", Icons.settings),
+    _NavigationItem(null, "Log Out", Icons.logout,
+        type: NavigationItemType.logout)
```

Comment:
z racji ze jestesmy w null safety to najlepiej byloby zastapic nulle jakimis '' itp